[2023-12-08 14:01:08,400] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2023-12-08T13:00:00+00:00 [queued]>
[2023-12-08 14:01:08,416] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2023-12-08T13:00:00+00:00 [queued]>
[2023-12-08 14:01:08,416] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-12-08 14:01:08,416] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-12-08 14:01:08,416] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-12-08 14:01:08,425] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2023-12-08T13:00:00+00:00
[2023-12-08 14:01:08,430] {standard_task_runner.py:52} INFO - Started process 501 to run task
[2023-12-08 14:01:08,432] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting', 'merge_data', '2023-12-08T13:00:00+00:00', '--job-id', '106', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpg2og9obe', '--error-file', '/tmp/tmpjn4457pf']
[2023-12-08 14:01:08,434] {standard_task_runner.py:77} INFO - Job 106: Subtask merge_data
[2023-12-08 14:01:08,473] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting.merge_data 2023-12-08T13:00:00+00:00 [running]> on host b4ba454d0948
[2023-12-08 14:01:08,522] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-08T13:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-08T13:00:00+00:00
[2023-12-08 14:01:08,547] {python.py:151} INFO - Done. Returned value was: None
[2023-12-08 14:01:08,557] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting, task_id=merge_data, execution_date=20231208T130000, start_date=20231208T140108, end_date=20231208T140108
[2023-12-08 14:01:08,604] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-12-08 14:01:08,625] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
